### PR TITLE
update "before you start" in quickstart

### PR DESCRIPTION
**Pull request by hisunnymay**
update "before you start" in reuse-component.md & build-module.md & build-provider.md

### CODE REVIEW

File: docs/guide/build-provider.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ In the previous guide, we showed you how to build an hours module including decl
 ## Before you start
 
 - [Sign up](./signingup.md#sign-up) for an account and log in to [OpenCUI](https://build.opencui.io/login).
-- [Build a simple chatbot](./build-simple-chatbot.md) for how to build a skill with entity slots.
-- [Reuse a full-stack module](./reuse-component.md) to get familiar with the functionality we try to build here.
+- Review the [example dialogue](./reuse-component.md#pick-a-full-stack-component) to get familiar with the functionality you will build here.
```
Comment (hisunnymay) [inline docs/guide/build-provider.md:17]: 如果我们同意：阅读该文档的人不需要阅读整个 build a simple chatbot 和 reuse a full stack component 的话，我可以之后再在 provider 中添加类似于 create project, import 等的截图

Comment (xiaoyunwu) [reply docs/guide/build-provider.md:17]: Before we spend time on this, which I think we eventually should. Can we have some common understanding on what is most pressing need when it comes to documentation? 
I think the idea should be, many things are broken, in fact there are infinite number of things are broken. But what are the most broken things right now? Is this the most broken thing in the entire website?

Comment (xiaoyunwu) [reply docs/guide/build-provider.md:17]: The point is: we need to get all the doc to 60 before we try to improve them to 80, then 90, and 95..., right now, my feeling are there are plenty of the doc are less than 60..., If you understand what I mean.

Comment (hisunnymay) [reply docs/guide/build-provider.md:17]: right, understood

**Comment by hisunnymay** [comment]
先 close 了，如果之后需要修改可以参考

**Pull request closed without merging**